Dockerfile

```diff
@@ -9,8 +9,11 @@ RUN ./gradlew --version
 FROM base AS build
 WORKDIR /usr/src/app
 COPY build.gradle build.gradle
+RUN true
```

Comment:
Each `RUN` creates a new image layer. So, I'm not a huge fan of this workaround. Looks like another option is to set `DOCKER_BUILDKIT=1` when building the docker image. Mind trying

```
DOCKER_BUILDKIT=1 ./docker/up.sh --seed
```

in the `.gitpod.yml`?